packages/components/src/MenuButton.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
+import React from 'react'
+
+import { IconButton, IconButtonProps } from './IconButton'
+import type { ForwardRef } from './types'
+
+export const MenuIcon = ({ size = 24 }) => {
```

Comment:
We should either export all our internal icons (Checkbox & Close) or document why. I think the Checkboxes are less useful outside the library but Close should be exported if this is

Reply:
Fair point.

Reply:
Fixed this by exporting `CloseIcon`!